compiler/rustc_middle/src/ty/region.rs

```diff
@@ -336,7 +336,9 @@ pub struct EarlyParamRegion {
 
 impl std::fmt::Debug for EarlyParamRegion {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{:?}, {}, {}", self.def_id, self.index, self.name)
+        // self.def_id goes first because of `erased-regions-in-hidden-ty.rs` being impossible to write
```

Comment:
make this a FIXME, this is not a good reason. I would expect the preferred repr to be name, index, `def_id`

Reply:
agreed idk why I didn't make this a FIXME